devicetypes/smartthings/zwave-door-window-sensor.src/zwave-door-window-sensor.groovy

```diff
@@ -78,6 +80,8 @@ def parse(String description) {
 }
 
 def updated() {
+	// Device-Watch simply pings if no device events received for 32min(checkInterval)
+	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
Should be similar to `fibaro-motion-sensor`. Should be set to 8 hrs and 2 minutes